Jimena/docs updates part 2 (#4285)
SwissDataScienceCenter/renku: docs/docs/10-users/60-sessions/guides/_100-css-integration.mdx

import { Columns, Column } from '@theme/MDXComponents';

# RenkuLab: CSCS integration

:::info
The RenkuLab: CSCS integration allows users to start interactive Renku sessions running as HPC jobs at CSCS.

RenkuLab accesses compute and storage resources on the behalf of the user, using existing permissions and resource allocation. Running Renku sessions at CSCS will consume compute credits from your CSCS account(s) and may affect other resources you have at CSCS (e.g. storage).

:::

# Instructions

### Prerequisites

1. You need to have an account at [CSCS](https://cscs.ch) which can launch jobs on the [HPC platform](https://docs.cscs.ch/platforms/hpcp/) or the [ML platform](https://docs.cscs.ch/platforms/mlp/).
2. You need to have a [RenkuLab.io](https://renkulab.io) account.

> ### Step 1: Connect your CSCS from RenkuLab

Reviewer: ```suggestion
### Step 1: Connect to the CSCS integration in RenkuLab
```

Go to the [integrations page](https://renkulab.io/integrations?targetProvider=cscs.ch) and connect with the CSCS integration. 

### Step 2: Complete the  [Request access to the Renku:CSCS Integration](https://www.notion.so/2810df2efafc803a8175ccd2d472b23d?pvs=21)

> Fill in the form so that a RenkuLab admin can let you launch sessions with the CSCS integration.

Reviewer: It's odd to have a link in a title like this. I suggest moving the link to the body text.

```suggestion
### Step 2: Complete the form

Fill in the [request access to the Renku/CSCS Integration form](https://www.notion.so/2810df2efafc803a8175ccd2d472b23d?pvs=21) so that a RenkuLab admin can let you launch sessions with the CSCS integration.
```

                    1. Create a session launcher with an **External Environment**
                    2. For the configuration, copy configuration from [this session launcher](https://staging.dev.renku.ch/p/flora.thiebaut/demo-hpc#launcher-01K5XTKBQK4XCCB6WTAQ7AVZJZ).
                    3. Ask to be added to a Daint resource pool
                    </p> 
                    Tested: has torch and reports cuda access to 4 GPUs
                    </details>

    - Code repositories added to the Renku project are added to your session

  </div>

  <div style={{ flex: 1, padding: "20px", border: '1px solid #ccc' }}>
    **Not working at the moment**

    <p>:x: Renku data connectors</p>
    <p>:x: External environments (bringing your own docker image)</p>
    <p>:x: Renku session secrets</p>
    <p>:x: Configuring the max run time of the job</p>
    <p>:x: Submitting slurm jobs from within the session</p>

  </div>
</div>

------ 

# How does this work? Where can I see my things at CSCS?

> In the Renku session logs, lookout for the following logs:

Reviewer: ```suggestion
In the Renku session logs, look out for the following logs:
```
